src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs

```diff
@@ -283,7 +283,7 @@ public async Task<HttpResponseMessage> SendAsync(CancellationToken cancellationT
                 if (cancellationToken.IsCancellationRequested)
                 {
                     _stream.AbortWrite((long)Http3ErrorCode.RequestCancelled);
-                    throw new OperationCanceledException(ex.Message, ex, cancellationToken);
+                    throw new TaskCanceledException(ex.Message, ex, cancellationToken);
```

Comment:
Why this change? Is it to unify exception types with H/2, H/1.1?

Reply:
Right, this is something that both 1.1 and 2 seem to be doing.
E.g. for 1.1:
https://github.com/dotnet/runtime/blob/dc9979136cf0ff1addbc1a4c0f06ea4b65f002d4/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs#L842

It seems harmless enough of a change (given TCE derives from OCE), or we could update the [failure tests](https://github.com/dotnet/runtime/blob/b3847f42ae95cc082a6b28575d4d847187b31a7b/src/libraries/System.Net.Http/tests/FunctionalTests/TelemetryTest.cs#L238) to expect OCE instead.